### PR TITLE
Refactor/html accessibility

### PR DESCRIPTION
**Pull request by tylrs**
# Description
Make HTML WAVE compliant remove redundant HTML, CSS, and JS

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -285,7 +286,7 @@ function renderPastActivities() {
     pastActivitiesCards.innerHTML =
     `
     <div class="no-activities-text">
-      <p>You haven't logged any activities yet.</p>
+      <p>You haven"t logged any activities yet.</p>
```
Comment (hoomberto) [inline main.js:289]: This needs to be single quote!

File: index.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (AshleyOh-bit) [inline index.html:2]: Good catch-I bet this cleared our WAV alerts!

File: index.html [hunk 3]
```diff
@@ -78,21 +78,11 @@ <h3>Select a category:</h3>
       <section class="past-activity">
         <h2>Past Activities</h2>
         <section class="activity-cards">
-          <div class="past-activity-card">
-            <div class="card-border">
-            </div>
-            <div class="card-text">
```
Comment (hoomberto) [inline index.html:84]: Probably best to delete this after all

**Review by hoomberto — APPROVED** [review]

File: main.js [hunk 4]
```diff
@@ -22,24 +22,25 @@ var activeMeditateIcon = document.querySelector("#activeMeditateIcon");
 var activeExerciseIcon = document.querySelector("#activeExerciseIcon");
 
 // Timer elements
-var currentView = document.querySelector('.current-activity');
-var timerContainer = document.querySelector('.current-activity');
-var timerNumbers = document.querySelector('.timer-numbers');
-var timerCategory = document.querySelector('#actual-timer-container h3');
-var startTimerButton = document.querySelector('.start-timer');
+var currentView = document.querySelector(".current-activity");
```
Comment (AshleyOh-bit) [inline main.js:25]: great attention to detail with the quoties!

**Review by AshleyOh-bit — APPROVED** [review]

**Pull request merged**